code.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+def lambda_handler(event, context):
+    if "body" not in event:
+        return "no body found"
+
+    # 1. Parse GitHub Webhook
+    payload = json.loads(event["body"])
```

Comment:
> [!WARNING]
> Using print() for logging in a Lambda function is not ideal. AWS Lambda logs to CloudWatch automatically when using proper logging.

> [!TIP]
> Use the logging module instead of print statements for better log management in production.

```python
import json
import logging

logger = logging.getLogger()
logger.setLevel(logging.INFO)

def lambda_handler(event, context):
    # ...
    payload = json.loads(event["body"])
    logger.info(f"Received webhook payload: {payload}")
```